docs/MISSION_AND_VALUES.md

```diff
@@ -0,0 +1,57 @@
+# Mission and Values
+
+This document is meant to help guide decisions about the future of `napari`, be it in terms of
+whether to accept new functionality, changes to the styling of the code or GUI, or whether to take on new dependencies,
+among other things. It serves as a point of reference for [core developers](CORE_DEV_GUIDE.md)
+actively working on the project, and an introduction for newcomers who want to learn a little more
+about where the project is going and what the teams values are. You can also learn more about how the project is managed by looking at our [governance model](GOVERNANCE).
+
+## Our mission
+
+napari aims to be the **multi-dimensional image viewer for Python** and to **provide GUI access to a plugin ecosystem of image analysis tools for scientists** to use in their daily work. We hope to accomplish this by:
+
+- being **easy to use and install**. We are careful in taking on new dependencies, sometimes making them optional, and will support a fully packaged installation that works cross-platform.
+
+- being **well documented** with **comprehensive tutorials and examples**. All functions in our API have thorough docstrings clarifying expected inputs and outputs, and we maintain a separate [tutorials and example website](http://napari.org) to explain different use cases and working modes.
+
+- providing **GUI access** to all critical functionality so napari can be used by people with no coding experience.
+
+- being **interactive** and **highly performant** in order to support very large data sets.
+
+- providing a **consistent and stable API** to enable plugin developers to build on top of napari without their code constantly breaking and to enable advanced users to build out sophisticated Python workflows.
```

Comment:
Perhaps provide a caveat about version numbers here? We don't require stability right now, but merely encourage it.

Reply:
added caveats, can put more in now, or can wait till next major release to put more detail in here